_posts/2018-03-28-puppeteer-tests.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Using Cockpit test VMs with your own test framework
+author: pitti
+date: 2018-03-28
+category: tutorial
+tags: cockpit starter-kit tests puppeteer
+slug: cockpit-custom-test-framework
+comments: true
+---
+
+The [Cockpit Starter Kit](https://github.com/cockpit-project/starter-kit/) provides the scaffolding for your own Cockpit
+extensions: a simple page (in React), build system (webpack, babel, eslint, etc.), and an integration test using
+Cockpit's own Python test API (on top of the
+[Chromium DevTools Protocol](https://chromedevtools.github.io/devtools-protocol/).  See the recent
```

Comment:
The parentheses starting at "on top of" is not closed after the link here.

Reply:
Curious, I already had this fixed in my local git, but didn't commit it. Thanks!

Reply:
Thanks!